### PR TITLE
Truncate whole words by char count

### PR DESCRIPTION
**Pull request by adamf321**

### CODE REVIEW

File: lib/chars.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+angular.module( 'lnFilters' )
+  .filter( 'chars', chars );
```
Comment (mitogh) [inline lib/chars.js]: Can we add a more meaningful name for this filter instead.

File: lib/chars.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+function chars() {
+  return function ( input, chars, breakOnWord ) {
+    if ( isNaN( chars ) ) return input;
+    if ( chars <= 0 ) return '';
+    if ( input && input.length > chars ) {
+      input = input.substring( 0, chars );
```
Comment (mitogh) [inline lib/chars.js]: Can we use `trim()` before to remove any extra space that might exist on the original input.

File: lib/chars.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+      if ( !breakOnWord ) {
```
Comment (mitogh) [inline lib/chars.js]: Can we invert the logic here to avoid as follows: 

```
if ( breakOnWord ) {
  // Do if here
} else {
  // otherwise here. 
} 
```

**Comment by adamf321** [comment]
Updated. The `breakOnWord` option wasn't actually necessary, as if you don't want to break on a word you can use the default angular `limitTo` filter.

**Pull request merged**